explorer/src/components/UniversalTable.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import { printableCoin } from '@nymproject/nym-validator-client';
+import { cellStyles } from './Universal-DataGrid';
+
+export type ColumnsType = {
+  field: string;
+  title: string;
+  headerAlign: string;
+  flex?: number;
+  width?: number;
+};
+
+export interface UniversalTableProps {
+  tableName: string;
+  columnsData: ColumnsType[];
+  rows: any[];
```

Comment:
'Need this to be typed. See comment below on how we could do this.

Reply:
hah yeah, beat you to it, sorry. i got there in the end.